README.md

```diff
@@ -1,29 +1,29 @@
 ## Headerless.org
 
-The Headerless Organisation committed itself to promote a streamlined license and copyright process in FOSS projects.
+The **Headerless Initiative** is committed to promote a streamlined license and copyright process in FOSS projects.
 
 ### Why Headerless
 
-The **F**ree and **O**pen **S**ource **S**oftware (FOSS) movement has created a prosperous and almost utopic ecosystem of sharing and collaborating. While the burden of tooling and services around creating, or contributing to FOSS projects has decreased dramatically over the last years, other topics such as licensing are complex to understand and represented itself as intransparent especially for beginners wanting to join the FOSS community. While permissive licenses such as the [MIT](https://opensource.org/licenses/MIT) and [Apache-2.0](https://opensource.org/licenses/Apache-2.0) are easy to understand and comply with, capturing the license situation of a project with mixed licenses on file level is overwhelming and discouraging.
+The **F**ree and **O**pen **S**ource **S**oftware (FOSS) movement has created a prosperous and almost utopistic ecosystem of sharing and collaborating. While the burden of tooling and services around creating or contributing to FOSS projects has decreased dramatically, topics such as licensing remain complex. Especially beginners wanting to join the FOSS community can be easily distracted by all the guidelines and practices around the handling of copyrights and licenses. While permissive licenses such as the [MIT](https://opensource.org/licenses/MIT) and [Apache-2.0](https://opensource.org/licenses/Apache-2.0) are easy to understand and comply with, capturing the license situation of a project with mixed licenses on file level is overwhelming and discouraging.
```

Comment:
Not really a finding but more of a question: what was the reason to change `utopic` to `utopistic`? As per oxford grammar, this would have ben correct: https://en.oxforddictionaries.com/definition/utopic

Reply:
The oxford comma at this point `creating, or` was actually proposed by Microsoft Word and would be grammatically correct.